### PR TITLE
CSS - Layout - Classes - Addy Abid

### PR DESCRIPTION
**Pull request by AddyAbid**
https://user-images.githubusercontent.com/85080021/130146779-ef25b068-5133-4331-a334-a984cf94a4f7.mp4

### CODE REVIEW

File: css-layout-classes/styles.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  width: 25%;
+}
+
+div.column-p {
+  width: 25%;
+}
+
+div.column-n {
+  width: 25%;
+}
```
Comment (codingcodymiller) [inline css-layout-classes/styles.css:49]: These column class names don't seem to have any meaning. They should be named after their size, such as `column-half`, `column-quarter`, `column-third`, etc.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by AddyAbid** [comment]
Screen Shot 2021-08-19 at 2 50 27 PM

File: css-layout-classes/index.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  <body>
+    <div class="container">
+      <h1>Choose your character!</h1>
+      <div class="row">
+        <div class="column">
+
+        </div>
+      </div>
+      <div class="row">
+        <div class="column">
```
Comment (codingcodymiller) [inline css-layout-classes/index.html:16]: `column` what? What's its size?

File: css-layout-classes/index.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+          </div>
+          <div class="column-second-half">
+          <img src="images/fox.png" alt="Fox">
+          </div>
+          <div class="column-three-fourths">
+            <img src="images/pikachu.png" alt="Pikachu">
+          </div>
+          <div class="column-fourth">
+            <img src="images/ness.png" alt="Ness">
+          </div>
```
Comment (codingcodymiller) [inline css-layout-classes/index.html:41]: These are all the same size, right? Why not use one `column-one-fourth` class for each of them?

File: css-layout-classes/index.html [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+        </div>
+        <div class="row">
+          <div class="column-half">
+            <h2>Donkey Kong</h2>
+          </div>
+        </div>
```
Comment (codingcodymiller) [inline css-layout-classes/index.html:47]: This shouldn't be its own row, it should be with the rest of the text about Donkey Kong

File: css-layout-classes/index.html [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+        <div class="row">
+          <div class="column-two-thirds">
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Obcaecati atque tenetur aperiam voluptate deserunt praesentium architecto assumenda incidunt modi. Doloribus adipisci dolorem magni quis minima, architecto repudiandae eius. Voluptate, dolor.</p>
+
+            <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Nostrum dolorum rem expedita molestias natus ipsum temporibus dolore, nobis vel asperiores similique animi itaque dolores quae cumque cum debitis, ea quia?</p>
+          </div>
+          <div class="column-right-d">
```
Comment (codingcodymiller) [inline css-layout-classes/index.html:54]: right-d? That doesn't tell me anything about the effect of the class 🤔

File: css-layout-classes/index.html [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+          </div>
+        </div>
+        <div class="row">
+          <div class="column-first">
+            <img src="images/mario.png" alt="Mario">
+          </div>
+          <div class="column-middle">
+            <h2>Mario</h2>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Blanditiis sint nemo accusamus totam ipsa eaque tempora mollitia iure. Voluptatum ratione consequuntur consequatur eos architecto velit corporis accusamus harum iure nostrum.</p>
+          </div>
```
Comment (codingcodymiller) [inline css-layout-classes/index.html:65]: These column classes still don't describe their behavior. What is their size? Name the class after that. `column-one-third`, `column-two-thirds`, etc.

File: css-layout-classes/styles.css [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+div.column-three-fourths {
+  width: 25%;
+}
+
+div.column-fourth {
+  width: 25%;
+}
```
Comment (codingcodymiller) [inline css-layout-classes/styles.css:49]: There should be no need for multiple classes with the exact same styling. Make one class with a good name describing its purpose, and then reuse that class.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by AddyAbid** [comment]
Screen Shot 2021-08-19 at 3 05 42 PM

**Review by codingcodymiller — APPROVED** [review]
MUCH better!!! 👏

**Pull request merged**